darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1579,7 +1579,7 @@ def save(self, path: Optional[str] = None) -> None:
             torch.save(self, f_out)
 
         # save the LightningModule checkpoint
-        path_ptl_ckpt = path + ".ckpt"
+        path_ptl_ckpt = path.replace(".pt", "") + ".ckpt"
```

Comment:
This change will break the logic in the `load()` method, I would rather not include it in this PR and discuss it in a separate issue.

Reply:
Agreed 👍  @SaeedForoutan, can you apply the changes?